apps/src/lib/levelbuilder/lesson-editor/ActivitySectionCardButtons.jsx

```diff
@@ -57,7 +57,10 @@ class ActivitySectionCardButtons extends Component {
     appendVocabularyLink: PropTypes.func.isRequired,
     appendSlide: PropTypes.func.isRequired,
     hasLessonPlan: PropTypes.bool.isRequired,
-    vocabularies: PropTypes.arrayOf(vocabularyShape).isRequired
+    vocabularies: PropTypes.arrayOf(vocabularyShape).isRequired,
+    allowMajorCurriculumChanges: PropTypes.bool.isRequired,
+    lastActivitySection: PropTypes.bool.isRequired,
+    lastActivity: PropTypes.bool.isRequired
```

Comment:
if it is not too much trouble, could you please name these so that they are more clearly booleans, i.e. `isLastActivity` and `isLastActivitySection`? the variable names starting with `has` and `allow` are great examples of this.

Reply:
Will do!